.github/workflows/build.yml

```diff
@@ -111,11 +111,12 @@ jobs:
         rm -f ~/.gradle/caches/modules-2/gc.properties
 
   ios:
-    runs-on: macos-latest
+    runs-on: macos-11
     name: iOS
     env:
       CCACHE_DIR: ${{ github.workspace }}/.ccache
       USE_CCACHE: 1
+      DEVELOPER_DIR: /Applications/Xcode_13.0.app/Contents/Developer
```

Comment:
This isn't pretty, but it's the [only way](https://www.jessesquires.com/blog/2020/01/06/selecting-an-xcode-version-on-github-ci/)